data/extract.py

```diff
@@ -25,6 +25,7 @@ def invoke(mock1, mock2):
   # called arguments are in `mock_setup.call_args`
   call_args = mock1.call_args or mock2.call_args
   args, kwargs = call_args
-  print(json.dumps(kwargs, indent=2))
+  with open('renovate-python-report.json', 'w', encoding='utf-8') as f:
+    json.dump(kwargs, f, ensure_ascii=False, indent=2)
```

Comment:
Do we need to clean this up afterwards?

Reply:
Don't think so, we do the same with gradle (no cleanup)